src/quo/components/wallet/keypair/view.cljs

```diff
@@ -91,23 +90,21 @@
   [account-list-card/view item])
 
 (defn- view-internal
-  [{:keys [default-selected?]}]
-  (let [selected? (reagent/atom default-selected?)]
-    (fn [{:keys [accounts action container-style] :as props}]
-      [rn/pressable
-       {:style    (merge (style/container (merge props {:selected? @selected?})) container-style)
-        :on-press #(when (= action :selector) (reset! selected? (not @selected?)))}
-       [rn/view {:style style/header-container}
-        [avatar props]
-        [rn/view
-         {:style {:margin-left 8
-                  :flex        1}}
-         [title-view (assoc props :selected? @selected?)]
-         [details-view props]]]
-       [rn/flat-list
-        {:data      accounts
-         :render-fn acc-list-card
-         :separator [rn/view {:style {:height 8}}]
-         :style     {:padding-horizontal 8}}]])))
+  [{:keys [accounts action container-style selected? on-press] :as props}]
+  [rn/pressable
+   {:style    (merge (style/container (merge props {:selected? selected?})) container-style)
+    :on-press #(when (= action :selector) (on-press))}
+   [rn/view {:style style/header-container}
+    [avatar props]
+    [rn/view
+     {:style {:margin-left 8
+              :flex        1}}
+     [title-view (assoc props :selected? selected?)]
+     [details-view props]]]
+   [rn/flat-list
+    {:data      accounts
+     :render-fn acc-list-card
+     :separator [rn/view {:style {:height 8}}]
+     :style     {:padding-horizontal 8}}]])
```

Comment:
I know most of this code is not added in this PR, but I see it a little dirty:

we should move the styles to the style ns and the code style for this:
```clojure
(merge (style/container (merge props {:selected? selected?})) container-style)
```

makes it hard to understand, we can change the `style/container` implementation and instead pass the params:
```clojure
(style/container props selected? container-style)
```